code/modules/ai/effect_node.dm

```diff
@@ -84,10 +98,10 @@ This means that the proc will pick out the *best* node
 
 	//If there's no adjacent nodes then let's throw a runtime (for mappers) and at admins (if they by any chance were spawning these in)
 	if(!length(adjacent_nodes))
-		message_admins("[ADMIN_VERBOSEJMP(src.loc)] was unable to connect to any considered-adjacent nodes; place them correctly if you were spawning these in otherwise report this.")
-		stack_trace("An ai node was initialized but no considered-adjacent nodes were nearby; this can be because of a mapping/admin spawning issue.")
+		message_admins("[ADMIN_VERBOSEJMP(src.loc)] was unable to connect to any considered-adjacent nodes; place them correctly if you were spawning these in, otherwise report this.")
+		stack_trace("An ai node was initialized but no considered-adjacent nodes were nearby; this can be because of a mapping/admin spawning issue. X Y Z coords is [src.x] [src.y] [src.z]")
```

Comment:
```suggestion
		stack_trace("An ai node was initialized but there were no considered-adjacent nodes nearby; this can be because of a mapping/admin spawning issue. X Y Z coords is [src.x] [src.y] [src.z]")
```